sdk/url_builders.go

```diff
@@ -0,0 +1,63 @@
+package sdk
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+const (
+	SystemSubDomain = "system"
+)
+
+func FormatEndpointURL(gatewayURL string, event *Event) (string, error) {
+	systemURL, formatErr := FormatSystemURL(gatewayURL)
+	if formatErr != nil {
+		return "", fmt.Errorf("error while formattig endpoint URL: %s", formatErr.Error())
+	}
+	personalURL := strings.Replace(systemURL, SystemSubDomain, event.Owner, -1)
+
+	return fmt.Sprintf("%s/%s", personalURL, event.Service), nil
+}
+
+func FormatDashboardURL(gatewayURL string, event *Event) (string, error) {
+	systemURL, formatErr := FormatSystemURL(gatewayURL)
+	if formatErr != nil {
+		return "", fmt.Errorf("error while formatting dashboard URL: %s", formatErr.Error())
+	}
+
+	return fmt.Sprintf("%s/dashboard/%s", systemURL, event.Owner), nil
+}
+
+func GetSubDomain(URL string) (string, error) {
+	parsedURL, parseErr := url.Parse(URL)
+	if parseErr != nil {
+		return "", fmt.Errorf("Unable to parse URL: %s", parseErr.Error())
+	}
+	subdomain := strings.Split(parsedURL.Host, ".")
+
+	//Host is www.world.org and subdomain would be www aka. 0th element of the slice
```

Comment:
Can you make this part of the godoc and also rename to `GetSubdomain`?

Reply:
Of course! 🍻